eslint.config.mjs

```diff
@@ -1,8 +1,8 @@
 import eslint from '@eslint/js'
 import tseslint from 'typescript-eslint'
 import globals from 'globals'
-import { readFileSync } from 'fs'
-import { resolve } from 'path'
+import { readFileSync } from 'node:fs'
```

Comment:
Is there an eslint rule we could apply/create to enforce this? Seems like a thing that we'll forget to do, like, *immediately*.